langwatch/prisma/migrations/20260802120003_webhook_delivery_indexes/migration.sql

```
@@ -0,0 +1,10 @@
+-- CreateIndex: the 30-day retention prune deletes by firedAt alone, and
+-- attempt grouping reads one batch's rows by (endpointId, dispatchId).
+CREATE INDEX "WebhookEndpointDelivery_firedAt_idx" ON "WebhookEndpointDelivery"("firedAt");
```

SQLFluff (4.2.2)

[error] CREATE INDEX should use CONCURRENTLY to avoid locking the table during the build.

(PG01)